core/src/main/scala/org/apache/spark/SparkHadoopConf.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark
+
+import java.net.URL
+
+import org.apache.hadoop.conf.Configuration
+
+import org.apache.spark.deploy.SparkHadoopUtil
+
+/**
+ * Hadoop Configuration for the Hadoop code (e.g. file systems).
+ */
+
+class SparkHadoopConf(conf: Configuration) {
+
+  def get: Configuration = conf
```

Comment:
We can just refer to `this.conf` ? Also, it's confusing to have `get()` and `get(name: String)` for completely different purpose.

Reply:
updated.